### DISCUSSION TITLE
How to use a lookback period?

### DISCUSSION THREAD

**Discussion by botradingblog1** [kernc/backtesting.py#664] [Q&A]
Hi,

just getting started with backtesting.py and it looks very promising.
I've been looking for a sample for using a lookback period.
Let's say I have a strategy that creates a signal when the MACD crosses a signal line. At the same time I want to check a certain RSI value in the last x days. How can access a range of RSI values during a lookback period?

Any examples greatly appreciated! Thanks!

**Comment by kernc** [comment]
Have you went through [the tutorials](https://kernc.github.io/backtesting.py/doc/backtesting/#tutorials)?
